src/App.tsx

```diff
@@ -136,15 +143,7 @@ function App(props: IAppProps) {
               ref={inputRef}
               onCreateTask={createNewTask}
             />
-            <div>
-              <TaskSection title="Today" query={getTodayTaskQuery()} />
-              <TaskSection title="Scheduled" query={getScheduledTaskQuery()} />
-              <TaskSection
-                title="Anytime"
-                query={getAnytimeTaskQuery()}
-                groupBy={GroupBy.Page}
-              />
-            </div>
+            <TaskCategoryTabSection tabs={categoryTabs}/>
```

Comment:
I think this is better, what do you think:

```
<TaskCategoryTabSection
  tabs={[
    { title: 'Today', query: getTodayTaskQuery() },
    // ...
  ]}
/>
```

Reply:
The advantage of this is that in the future it will be possible to directly allow users to customize the category config.